pre-docs/running_bolt_commands.md

```diff
@@ -38,6 +38,13 @@ bolt command run "netstat -an | grep 'tcp.*LISTEN'" --nodes web5.mydomain.edu, w
 
     **Note:** When connecting to Bolt hosts over WinRM that have not configured SSL for port 5986, passing the `--no-ssl` switch is required to connect to the default WinRM port 5985.
 
+    **Warning:** When running shell one-liners that include redirection or
```

Comment:
```suggestion
## Running commands with redirection or pipes

When you run one-line commands that include redirection or pipes, pass `bash` or another shell as the command. Using a shall ensures that the one-liner is run as a single command and that it works correctly with `run-as`. For example, instead of `bolt command run "echo foo > /root/foo" --run-as root`, use `bolt command run "bash 'echo foo > /root/foo'" --run-as root`.
```